src/scripts/components/ProfileCuration.js

```diff
@@ -0,0 +1,522 @@
+      : props.theme.colors.ProfileCuration.ChallengeBackgroundOneFrom}
+      0%,
+    ${props =>
+    props.inReveal
+      ? props.theme.colors.ProfileCuration.ChallengeBackgroundTwoTo
+      : props.theme.colors.ProfileCuration.ChallengeBackgroundOneTo}
+      100%
+  );
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-bottom: ${INFOSTATUS_CARD_MARGIN_BOTTOM};
+  padding: 50px 25px 60px;
+`
+
+const ChallengeTimeWrapper = styled.div`
+  margin: 35px 0;
+  text-align: center;
+  width: 100%;
+`
+
+const ChallengeTime = styled.div`
+  display: inline-flex;
+  flex-direction: column;
+  text-align: center;
+  vertical-align: bottom;
+  width: 16%;
+`
+
+const ChallengeSequence = styled(ChallengeTimeWrapper)`
+  margin: 0;
+`
+
+const ChallengeSequenceText = styled(Text)`
+  display: inline-block;
+  text-align: center;
+  width: 33.2%;
+`
+
+const ChallengeTimeline = styled(ChallengeTimeWrapper)`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  margin: 20px 0 0;
+  position: relative;
+  width: 70%;
+`
+
+const ChallengeLine = styled.span`
+  border-top: 2px ${({ dashed }) => (dashed ? 'dashed' : 'solid')}
+    ${props => props.theme.colors.ProfileCuration.ChallengeSequenceDot};
+  left: 0;
+  opacity: ${({ dashed }) => (dashed ? 0.5 : null)};
+  opacity: ${({ dashed }) => (dashed ? 0.5 : null)};
+  position: absolute;
+  top: 50%;
+  transform: translate3d(0, -50%, 0);
+  width: ${({ width }) => width || '100%'};
+`
+
+const DotActive = css`
+  &::before,
+  &::after {
+    content: '';
+  }
+`
+
+const ChallengeSequenceDot = styled.span`
+  background: ${props =>
+    props.theme.colors.ProfileCuration.ChallengeSequenceDot};
+  border-radius: 100%;
+  height: ${props => (props.active ? '12px' : '8px')};
+  position: relative;
+  width: ${props => (props.active ? '12px' : '8px')};
+  z-index: 2;
+  ${props => (props.active ? DotActive : null)} &::before, &::after {
+    border: 2px solid
+      ${props => props.theme.colors.ProfileCuration.ChallengeSequenceDot};
+    border-radius: 100%;
+    left: 50%;
+    position: absolute;
+    top: 50%;
+    transform: translate3d(-50%, -50%, 0);
+  }
+  &::before {
+    height: 22px;
+    opacity: 0.5;
+    width: 22px;
+  }
+  &::after {
+    height: 36px;
+    opacity: 0.2;
+    width: 36px;
+  }
+`
+
+class Profile extends Component<Props, void> {
+  render () {
+    const MockAvatar = (
+      <img src="http://www.onrouteforward.com/uploads/6/0/0/8/60080487/6808024.jpg" />
+    )
+    return (
+      <Wrapper>
+        <VideoWrapper>
+          <Text>Video</Text>
+          <VideoInfo nobackground>
+            <VideoInfoHeader>
+              <VideoInfoTitle small bold>
+                Video title
+              </VideoInfoTitle>
+              <VideoInfoHeaderContent>
+                <UserBadge
+                  userAvatar={MockAvatar}
+                  userName="Susan Medelin"
+                  userDate="Since 2018"
```

Comment:
Why hard-coded values here?

Reply:
The whole component is a mockup, @bent0b0x. It should be broken up into other components.
I'll remove it from App routes.

Reply:
ProfileCuration out of this **PR**